packages/kit/src/views/Prime/pages/PrimeDashboard/PrimeBenefitsList.tsx

```
@@ -49,6 +50,8 @@ function PrimeBenefitsItem({
 export function PrimeBenefitsList() {
   const navigation = useAppNavigation();
   const intl = useIntl();
+  const { ensureOneKeyIDLoggedIn } = usePrimeRequirements();
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)